pom.xml

```diff
@@ -87,6 +87,19 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>cobertura-maven-plugin</artifactId>
```

Comment:
Other comparison sources: https://answers.atlassian.com/questions/9880767/clover-vs-cobertura-jacoco-and-other-code-coverage-tools